packages/solid/src/components/Form.tsx

```diff
@@ -2,12 +2,17 @@ import { JSX, splitProps } from 'solid-js';
 import { handleSubmit } from '../methods/handleSubmit';
 import { FieldValues, FormState, SubmitEvent } from '../types';
 
+export type FormSubmitHandler<TFieldValues extends FieldValues> = (
```

Comment:
I would move this type to `types/form.ts` directly under `SubmitEvent` because it is also needed in `methods/handleSubmit.ts` in line 24. Furthermore, I would rename it to `SubmitHandler` so that the name is a bit shorter. Please also add a comment here describing the type.

Reply:
Moved it to `form.ts`, renamed and added it to SubmitHandler, also added the comment to describe what is.